libs/components/action-bars/src/lib/modules/summary-action-bar/fixtures/summary-action-bar.component.fixture.html

```
@@ -31,191 +31,192 @@
-      @if (extraActions) {
-      <sky-summary-action-bar-secondary-action
-        [disabled]="disableButtons"
-        (actionClick)="clickHandler()"
-      >
-        Secondary action 3
-      </sky-summary-action-bar-secondary-action>
-      <sky-summary-action-bar-secondary-action
+  </sky-summary-action-bar>
+}
+@if (showSecondaryActionBar) {
+  <sky-summary-action-bar id="summary-action-bar">
```

HTMLHint (1.9.2)

[error] The id value [ summary-action-bar ] must be unique.

(id-unique)